accurest-core/src/main/groovy/io/coderate/accurest/builder/SpockMethodBodyBuilder.groovy

```diff
@@ -18,8 +18,10 @@ class SpockMethodBodyBuilder {
 		blockBuilder.addLine('given:').startBlock()
 		blockBuilder.addLine('def request = given()')
 		blockBuilder.indent()
-		stubDefinition.request.headers.headers.collectEntries { [(it.name): it.serverValue] }.each { Map.Entry entry ->
-			blockBuilder.addLine(".header('${entry.key}', '${entry.value}')")
+		if (stubDefinition.request.headers) {
+			stubDefinition.request.headers.headers.collectEntries { [(it.name): it.serverValue] }.each { Map.Entry entry ->
```

Comment:
you would have to add more ? that's why I'd prefer to add expiclit check with the if. IMO it's more readable